src/room/useGroupCall.ts

```diff
@@ -133,6 +133,7 @@ function getParticipants(
             : ConnectionState.WaitMedia
           : ConnectionState.EstablishingCall;
       }
+      console.log(`${member.userId} conn state is ${connectionState}`);
```

Comment:
This should go through `logger`

Reply:
Oops, actually didn't mean to commit that at all.